app/controllers/dashboard_controller.rb

```diff
@@ -2,5 +2,8 @@ class DashboardController < ApplicationController
   before_action :authorize!
 
   def show
```

Comment:
what do you think about this flow - eventually, I will change the ```@query.followers``` to a PORO request to pass to the view.